src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import Home from "./pages/Home";
+import Login from "./pages/Login";
+import Signup from "./pages/Signup";
+import Items from "./pages/Items";
+import Layout from "./layouts";
+
+const root = ReactDOM.createRoot(document.getElementById("root"));
+
+root.render(
+  <BrowserRouter>
+    <App>
+      <Routes>
+        <Route element={<Layout authed={true} />}>
+          <Route path="/" element={<Home />} />
+          <Route path="/items" element={<Items />} />
+        </Route>
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
+      </Routes>
+    </App>
+  </BrowserRouter>
```

Comment:
```suggestion
  <BrowserRouter>
    <App>
      <Routes>
        <Route element={<Layout authed={true} />}>
          <Route path="/" element={<Home />} />
          <Route path="/items" element={<Items />} />
        </Route>
        <Route path="/login" element={<Login />} />
        <Route path="/signup" element={<Signup />} />
      </Routes>
    </App>
  </BrowserRouter>
```

이부분을 root경로에 Router.jsx 컴포넌트로 관리해보세요!

Reply:
따로 만들 생각을 안했네요 router로 빼야겠습니다